### PR TITLE
(NFC) Bring up API folder to style of future coder checker

### PR DESCRIPTION
**Pull request by seamuslee001**
Overview
----------------------------------------
This brings up the API folder style to that of a newer drupal/coder version

ping @totten

### CODE REVIEW

File: api/v3/CxnApp.php [hunk 1]
```diff
@@ -25,8 +25,8 @@
  +--------------------------------------------------------------------+
  */
 
-use \Civi\Cxn\Rpc\Message\AppMetasMessage;
-use \Civi\Cxn\Rpc\Message\GarbledMessage;
+use Civi\Cxn\Rpc\Message\AppMetasMessage;
```
Comment (seamuslee001) [inline api/v3/CxnApp.php]: @totten you may have a feeling on this but i think this is probably ok

File: api/v3/Generic/Setvalue.php [hunk 2]
```diff
@@ -57,8 +57,8 @@ function civicrm_api3_generic_setValue($apiRequest) {
   $fields = civicrm_api($entity, 'getFields', [
     'version' => 3,
     'action' => 'create',
-    "sequential"]
-  );
+    'sequential' => 1,
```
Comment (seamuslee001) [inline api/v3/Generic/Setvalue.php]: @eileenmcnaughton seems this was very odd for a while

Comment (colemanw) [reply api/v3/Generic/Setvalue.php]: We should be careful here as fixing the incorrect syntax will change the output format by stripping the array keys. And looking farther down it seems to be relying on those array keys.

Comment (colemanw) [reply api/v3/Generic/Setvalue.php]: Looks like "fixing" this line caused broken tests. I recommend removing it instead.

**Comment by colemanw** [comment]
@seamuslee001 I've removed the problematic "sequential" param from `Setvalue.php` which was incorrect and never actually did anything.

File: api/v3/CxnApp.php (hunk 1 above)
Comment (totten) [reply api/v3/CxnApp.php]: Yeah, I think it's OK.

**Pull request merged**